proxy/src/telemetry/metrics/labels.rs

```diff
@@ -88,6 +125,153 @@ pub struct DstLabels {
+        85  => ERESTART       ,  // Interrupted system call should be restarted
+        86  => ESTRPIPE       ,  // Streams pipe error
+        87  => EUSERS         ,  // Too many users
+        88  => ENOTSOCK       ,  // Socket operation on non-socket
+        89  => EDESTADDRREQ   ,  // Destination address required
+        90  => EMSGSIZE       ,  // Message too long
+        91  => EPROTOTYPE     ,  // Protocol wrong type for socket
+        92  => ENOPROTOOPT    ,  // Protocol not available
+        93  => EPROTONOSUPPORT,  // Protocol not supported
+        94  => ESOCKTNOSUPPORT,  // Socket type not supported
+        95  => EOPNOTSUPP     ,  // Operation not supported on transport endpoint
+        96  => EPFNOSUPPORT   ,  // Protocol family not supported
+        97  => EAFNOSUPPORT   ,  // Address family not supported by protocol
+        98  => EADDRINUSE     ,  // Address already in use
+        99  => EADDRNOTAVAIL  ,  // Cannot assign requested address
+        100 => ENETDOWN       ,  // Network is down
+        101 => ENETUNREACH    ,  // Network is unreachable
+        102 => ENETRESET      ,  // Network dropped connection because of reset
+        103 => ECONNABORTED   ,  // Software caused connection abort
+        104 => ECONNRESET     ,  // Connection reset by peer
+        105 => ENOBUFS        ,  // No buffer space available
+        106 => EISCONN        ,  // Transport endpoint is already connected
+        107 => ENOTCONN       ,  // Transport endpoint is not connected
+        108 => ESHUTDOWN      ,  // Cannot send after transport endpoint shutdown
+        109 => ETOOMANYREFS   ,  // Too many references: cannot splice
+        110 => ETIMEDOUT      ,  // Connection timed out
+        111 => ECONNREFUSED   ,  // Connection refused
+        112 => EHOSTDOWN      ,  // Host is down
+        113 => EHOSTUNREACH   ,  // No route to host
+        114 => EALREADY       ,  // Operation already in progress
+        115 => EINPROGRESS    ,  // Operation now in progress
+        116 => ESTALE         ,  // Stale NFS file handle
+        117 => EUCLEAN        ,  // Structure needs cleaning
+        118 => ENOTNAM        ,  // Not a XENIX named type file
+        119 => ENAVAIL        ,  // No XENIX semaphores available
+        120 => EISNAM         ,  // Is a named type file
+        121 => EREMOTEIO      ,  // Remote I/O error
+        122 => EDQUOT         ,  // Quota exceeded
+        123 => ENOMEDIUM      ,  // No medium found
+        124 => EMEDIUMTYPE    ,  // Wrong medium type
+        125 => ECANCELED      ,  // Operation Canceled
+        126 => ENOKEY         ,  // Required key not available
+        127 => EKEYEXPIRED    ,  // Key has expired
+        128 => EKEYREVOKED    ,  // Key has been revoked
+        129 => EKEYREJECTED   ,  // Key was rejected by service
+        130 => EOWNERDEAD     ,  // Owner died
+        131 => ENOTRECOVERABLE,  // State not recoverable
```

Comment:
Why not have a default variant that just stores the error code? It seems wrong to lose the error code just because it isn't listed above.